libraries/chain/include/eosio/chain/types.hpp

```diff
@@ -132,6 +132,52 @@ namespace eosio { namespace chain {
          {}
    };
 
+   /**
+    * Compare vector, string, string_view, shared_blob (unsigned)
+    */
+   template<typename A, typename B>
+   inline int compare_blob(const A& a, const B& b) {
+      static_assert(
+         std::is_same_v<std::decay_t<decltype(*a.data())>, char> ||
+         std::is_same_v<std::decay_t<decltype(*a.data())>, unsigned char>);
+      static_assert(
+         std::is_same_v<std::decay_t<decltype(*b.data())>, char> ||
+         std::is_same_v<std::decay_t<decltype(*b.data())>, unsigned char>);
+
+      auto r = std::char_traits<unsigned char>::compare(
```

Comment:
The standard only defines specializations char_traits for char, [char8_t], char16_t, char32_t, and wchar_t.

Reply:
Switched to `memcmp`